ui/client/containers/ExternalLib/ExternalModule.tsx

```diff
@@ -0,0 +1,28 @@
+import {lazy} from "@loadable/component"
+import React, {PropsWithChildren, useMemo} from "react"
+import {LibContextProvider} from "./store"
+import {createScript, loadComponent, splitUrl} from "./tools"
+import {Module, ModuleUrl} from "./types"
+
+export function ExternalModule<M extends Module>(props: PropsWithChildren<{url: ModuleUrl}>): JSX.Element {
```

Comment:
Do I understand correctly, that this is central point of accessing module federation? Can we add some comment to this class? E.g. this is the place to look for if you want to use externalmodules in NK, see remoteTab for example